### PR TITLE
Readme instruction improvement

### PR DESCRIPTION
**Pull request by francis2054**
Edit with clearer instructions for newcomers

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -5,12 +5,13 @@
 Requires a connected dev board running Marauder FW. [See install instructions from UberGuidoZ here.](https://github.com/UberGuidoZ/Flipper/tree/main/Wifi_DevBoard#marauder-install-information)
 
 ## Get the app
-1. Navigate to the [FAP Build](https://github.com/0xchocolate/flipperzero-wifi-marauder/actions/workflows/build.yml)
-   GitHub action workflow, and select the most recent run.
-2. The FAP is built for the `dev` and `release` channels of both official and unleashed
+1. Make sure you're logged in with a github account (otherwise the downloads in step 2 wont work)
```
Comment (0xchocolate) [inline ReadMe.md]: can you change "wont" to "won't"?

File: ReadMe.md [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
+2. Navigate to the [FAP Build](https://github.com/0xchocolate/flipperzero-wifi-marauder/actions/workflows/build.yml)
+   GitHub action workflow, and select the most recent run, scroll down to artifacts.
+3. The FAP is built for the `dev` and `release` channels of both official and unleashed
    firmware. Download the artifact corresponding to your firmware version.
-3. Extract `ESP32_WiFi_Marauder.fap` from the ZIP file to your Flipper
-   Zero SD card.
+4. Extract `ESP32_WiFi_Marauder.fap` from the ZIP file to your Flipper
+   Zero SD card, preferably under Apps/GPIO along the rest of the GPIO apps.
```
Comment (0xchocolate) [inline ReadMe.md]: can you change "along" to "along with"?

**Review by 0xchocolate — COMMENTED** [review]
Just a couple grammar nitpicks

**Review by francis2054 — COMMENTED** [review]
Grammar corrections

**Review by 0xchocolate — APPROVED** [review]

**Pull request merged**